app/util/data_preparation/jira/prepare-data.py

```diff
@@ -77,36 +84,61 @@ def __write_to_file(file_path, items):
             f.write(f"{item}\n")
 
 
+def __create_data_set(jira_api):
+    dataset = dict()
+    dataset[USERS] = __get_users(jira_api)
+    software_project_keys = __get_software_project_keys(jira_api)
+    dataset[PROJECT_KEYS] = software_project_keys
+    dataset[ISSUES] = __get_issues(jira_api, software_project_keys)
+    dataset[SCRUM_BOARDS] = __get_boards(jira_api, 'scrum')
+    dataset[KANBAN_BOARDS] = __get_boards(jira_api, 'kanban')
+    dataset[JQLS] = __generate_jqls(count=150)
+
+    return dataset
+
+
+def __get_issues(jira_api, software_project_keys):
+    return jira_api.issues_search(
+        jql=f"project in ({','.join(software_project_keys)}) AND status != Closed order by key", max_results=8000)
+
+
+def __get_boards(jira_api, board_type):
+    return jira_api.get_boards(board_type=board_type, max_results=250)
+
+
+def __get_users(jira_api):
+    perf_users = jira_api.get_users(username=DEFAULT_USER_PREFIX, max_results=performance_users_count)
+    return generate_perf_users(api=jira_api, cur_perf_user=perf_users)
+
+
+def __get_software_project_keys(jira_api):
+    all_projects = jira_api.get_all_projects()
+    software_project_keys = [project['key'] for project in all_projects if 'software' == project.get('projectTypeKey')]
+
+    if not software_project_keys:
```

Comment:
It would be great to add simiral checks if scrum/kanban board is exists.

Reply:
Done